bookmarks/images/signals.py

```diff
@@ -0,0 +1,9 @@
+from django.db.models.signals import m2m_changed
+from django.dispatch import receiver
+from .models import Image
+
+
+@receiver(m2m_changed, sender=Image.users_like.through)
+def users_like_changed(sender, instance, **kwargs):
+    instance.total_likes = instance.users_like.count()
+    instance.save()

```

Comment:
Add a test for this. On adding a user like to an image, the total_likes should be updated